bundler.d/assets.rb

```diff
@@ -8,3 +8,4 @@
 gem 'sass-rails', '~> 5.0'
 gem 'spice-html5-rails', '~> 0.1.5'
 gem 'jquery_pwstrength_bootstrap_4', :require => 'jquery_pwstrength_bootstrap'
+gem 'webmks_sdk-rails', :git => 'https://github.com/timogoebel/webmks_sdk-rails.git'
```

Comment:
@ohadlevy: I wasn't sure about the license used. That's why I didn't want to push this to rubygems.
I'm no expert in software licenses. How can we make sure, this works? VMWare just has a EULA that you need to accept to download the software. The text is in the gem.